CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* **BREAKING** Allow product name in layout header component when public ([PR #1736](https://github.com/alphagov/govuk_publishing_components/pull/1736))
```

Comment:
I wonder if this doesn't quite reflect the breaking nature of this change - which are what I'd want to get my attention when upgrading. Perhaps: Layout header component always displays product name and environment when provided ?

Reply:
Good suggestion, thanks. Was a bit of a hangover from the original intent of the PR, have changed.